storage/persistent_storage/persistent-storage-aws.adoc

```diff
@@ -26,16 +26,12 @@ storage provider.
 
 .Additional References
```

Comment:
So here's a question; Does this make sense for the entire assembly or not? I ask because I've been using assembly level additional resources with a `.`, but converted those to `==` and now it appears in the TOC, which could be useful maybe? Just a thought.

Reply:
It does make sense for the entire assembly, yes. This particular reference is an odd use case, and as such, I also considered putting it in a note block. Interesting thought behind using `==` so it appears in the TOC. I'll try that out!